pyproject.toml

```diff
@@ -24,6 +24,10 @@ files = [
     "celery/apps/beat.py",
 ]
 
+[tool.codespell]
+ignore-words-list = "assertin"
+skip = "./docs/AUTHORS.txt,./docs/history/*,./docs/spelling_wordlist.txt,Changelog.rst,CONTRIBUTORS.txt,*.key"
```

Comment:
On localhost, `pre-commit run --all-files` passes all tests.

On GitHub Actions, line 28 is being obeyed but line 29 is being ignored!

Reply:
In `skip` Remove the leading `./` so `./docs/AUTHORS.txt` --> `docs/AUTHORS.txt`